appveyor.yml

```diff
@@ -32,44 +32,53 @@
     - '%AppData%\npm-cache'
 
   environment:
+    # https://github.com/jasongin/nvs/blob/master/doc/CI.md
+    NVS_VERSION: 1.4.0
     SKIP_SASS_BINARY_DOWNLOAD_FOR_CI: true
     matrix:
-      - nodejs_version: 0.10
+      - NODEJS_VERSION: node/0.10
         GYP_MSVS_VERSION: 2013
         APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
-      - nodejs_version: 0.12
+      - NODEJS_VERSION: node/0.12
         GYP_MSVS_VERSION: 2013
         APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
-      - nodejs_version: 1.0
+      - NODEJS_VERSION: iojs/1
         GYP_MSVS_VERSION: 2013
         APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
-      - nodejs_version: 1
+      - NODEJS_VERSION: iojs/2
         GYP_MSVS_VERSION: 2013
         APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
-      - nodejs_version: 2
+      - NODEJS_VERSION: iojs/3
         GYP_MSVS_VERSION: 2013
         APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
-      - nodejs_version: 3
+      - NODEJS_VERSION: node/4
         GYP_MSVS_VERSION: 2013
         APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
-      - nodejs_version: 4
+      - NODEJS_VERSION: node/5
         GYP_MSVS_VERSION: 2013
         APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
-      - nodejs_version: 5
-        GYP_MSVS_VERSION: 2013
-        APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
-      - nodejs_version: 6
+      - NODEJS_VERSION: node/6
         GYP_MSVS_VERSION: 2015
         APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
-      - nodejs_version: 7
+      - NODEJS_VERSION: node/7
         GYP_MSVS_VERSION: 2015
         APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
-      - nodejs_version: 8
+      - NODEJS_VERSION: node/8
         GYP_MSVS_VERSION: 2015
         APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      - NODEJS_VERSION: chakracore/8.1.2
```

Comment:
WASM is supported in Node-ChakraCore if you're using the WebAssembly methods from node. Using basic.wasm from [here](https://github.com/Microsoft/ChakraCore/blob/master/test/wasm/basic.wasm), the following code worked with Node-ChakraCore:
```
const fs = require('fs');
const buf = fs.readFileSync('basic.wasm')

async function test() {
    try {
        const module = await WebAssembly.compile(buf);
        const inst = new WebAssembly.Instance(module, {test: {foo: function(a){console.log(`foo called: ${a}`); return 2;}}});
        console.log(inst.exports.a(1));
    } catch (reason) { 
        console.log(`Failed: ${reason}`)
    }
}

test();
```

Note that ChakraCore doesn't expose JSRT APIs for embedders to directly invoke WASM from their applications- the workaround is they'll have to do it through JavaScript.

Also, minor technical point but while node-chakracore was included in the N-API repo during N-API prototyping, N-API has merged into the master branch of nodejs/node, so active N-API development for Node-ChakraCore is now happening in the master branch of nodejs/node-chakracore. But the meta point still stands that node-chakracore supports N-API.